### PR TITLE
Feature/vh/cli

### LINKED ISSUE #14: Add CLI counterpart to SDK notebooks
Add CLI counterpart to the solution accelerator. This would implement the same core functionality as the existing Python SDK v2 notebooks, but serve as good reference to for CLI first approach.

### PR DESCRIPTION
**Pull request by vhoudebine**
closes #14

### CODE REVIEW

File: README.md
Comment (zso3n3n) [inline README.md:1]: Can you update cli_pipelines link on line to the directory instead of readme2

File: cli_pipelines/1_training_pipeline.yml
Comment (zso3n3n) [inline cli_pipelines/1_training_pipeline.yml:1]: To keep things consistent across SDK and CLI what do you think about keeping the compute and data assets consistent.

Can you change the default compute and the data sources in the training, inference, and feedback pipelines to match the compute / data assets setup in the 0_setup notebook?

The workflow would then be 'Run setup notebook' --> run cli/sdk steps

File: README.md
Comment (zso3n3n) [inline README.md:1]: Can you add instructions on how to initialize the cli to AML workspace? Could put them here or in a separate readme in the cli_piplines dir. Also, add a sample config.json to the home dir.

**Review by zso3n3n — CHANGES_REQUESTED** [review]
Got everything to run, nicely done! Just have these three small comments requesting changes.

File: README.md
Comment (vhoudebine) [reply README.md:1]: Done, added the az configure instruction + how to overwrite in each CLI command

Comment (vhoudebine) [reply README.md:1]: done

File: cli_pipelines/1_training_pipeline.yml
Comment (vhoudebine) [reply cli_pipelines/1_training_pipeline.yml:1]: Not sure about this one, it seems kind of clunky to have people use the SDK and the CLI within the same workflow. I added instructions in the readme on how to create a cluster with the CLI. The data gets uploaded automatically when running CLI job creation so not sure we need to create data assets here. What do you think?

**Review by zso3n3n — APPROVED** [review]
LGTM. Nice job!

**Review by zso3n3n — APPROVED** [review]

**Pull request merged**